bin/Data/Scripts/Editor/EditorGizmo.as

```diff
@@ -215,7 +215,7 @@ void UseGizmo()
         return;
     }
 
-    IntVector2 pos = ui.cursorPosition;
+    IntVector2 pos = ui.ConvertSystemToUI(ui.cursorPosition);

```

Comment:
cursorPosition already should be in ui coords. If u need real mouse pos, then we have input->GetMousePosition